src/views/frontPage.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const FrontPage = (props) => (
```

Comment:
There is no need to add that props param since you are not using it anywhere in this component.